tests/epyccel/test_bitwise.py

```diff
@@ -0,0 +1,145 @@
+import pytest
+import numpy as np
+
+from pyccel.epyccel import epyccel
+from modules import bitwise
+from conftest import *
+
+@pytest.mark.parametrize("a, b",[(True, False),(True, True)])
+def test_right_shift_b_b(language, a, b):
+    f1 = bitwise.right_shift_b_b
+    f2 = epyccel( f1, language = language, verbose = True )
+    assert f1(a, b) == f2(a, b)
+    assert(type(f1(a, b)) == type(f2(a, b))) # pylint: disable=unidiomatic-typecheck
+
+@pytest.mark.parametrize("a, b",[(1, 1),(1, 2)])
+def test_right_shift_i_i(language, a, b):
+    f1 = bitwise.right_shift_i_i
+    f2 = epyccel( f1, language = language, verbose = True )
+    assert f1(a, b) == f2(a, b)
+    assert(type(f1(a, b)) == type(f2(a, b))) # pylint: disable=unidiomatic-typecheck
+
+@pytest.mark.parametrize("a, b",[(True, 2),(True, 1)])
+def test_right_shift_b_i(language, a, b):
+    f1 = bitwise.right_shift_b_i
+    f2 = epyccel( f1, language = language, verbose = True )
+    assert f1(a, b) == f2(a, b)
+    assert(type(f1(a, b)) == type(f2(a, b))) # pylint: disable=unidiomatic-typecheck
+
+@pytest.mark.parametrize("a, b",[(True, 1),(True, 2)])
+def test_left_shift_b_i(language, a, b):
+    f1 = bitwise.right_shift_b_i
+    f2 = epyccel( f1, language = language, verbose = True )
+    assert f1(a, b) == f2(a, b)
+    assert(type(f1(a, b)) == type(f2(a, b))) # pylint: disable=unidiomatic-typecheck
+
+@pytest.mark.parametrize("a, b",[(1, 1),(1, 2)])
+def test_left_shift_i_i(language, a, b):
+    f1 = bitwise.right_shift_i_i
+    f2 = epyccel( f1, language = language, verbose = True )
+    assert f1(a, b) == f2(a, b)
+    assert(type(f1(a, b)) == type(f2(a, b))) # pylint: disable=unidiomatic-typecheck
+
+@pytest.mark.parametrize("a, b",[(True, False),(True, True)])
+def test_left_shift_b_b(language, a, b):
+    f1 = bitwise.right_shift_b_b
+    f2 = epyccel( f1, language = language, verbose = True )
+    assert f1(a, b) == f2(a, b)
+    assert(type(f1(a, b)) == type(f2(a, b))) # pylint: disable=unidiomatic-typecheck
+
+@pytest.mark.parametrize("language",['c',pytest.param('fortran', marks = pytest.mark.xfail(reason="problem in wrapping"))])
```

Comment:
Please open an issue for this problem

Reply:
Ah sorry I have just realised that this is part of issue #370 . Can you reference the issue in the xfail message please